features/trusted-types.yml

```diff
@@ -31,5 +31,5 @@ compat_features:
   - api.TrustedTypePolicyFactory.isScript
   - api.TrustedTypePolicyFactory.isScriptURL
   - api.trustedTypes
-  - http.headers.Content-Security-Policy.require-trusted-types-for
-  - http.headers.Content-Security-Policy.trusted-types
+  # - http.headers.Content-Security-Policy.require-trusted-types-for
```

Comment:
I had to comment these out because `http.headers.Content-Security-Policy` has non-real values.

Reply:
Fixed by https://github.com/mdn/browser-compat-data/pull/23352 it seems.